scripts/utils/run_compareScenarios2.R

```diff
@@ -20,10 +20,16 @@ run_compareScenarios2 <- function(outputdirs, shortTerm, outfilename, regionList
   scenNames <- getScenNames(outputdirs)
   # Add '../' in front of the paths as compareScenarios2() will be run in individual temporary subfolders (see below).
   mif_path  <- file.path("..", outputdirs, paste("REMIND_generic_", scenNames, ".mif", sep = ""))
+  mif_path_polCosts  <- file.path("..", outputdirs, paste("REMIND_generic_", scenNames, "_adjustedPolicyCosts.mif", sep = ""))
   hist_path <- file.path("..", outputdirs[1], "historical.mif")
   scen_config_path  <- file.path("..", outputdirs, "config.Rdata")
   default_config_path  <- file.path("..", "config", "default.cfg")
 
+  # Use adjustedPolicyCosts mif, if available
+  if (file.exists(mif_path_polCosts)) {
```

Comment:
`mif_path_polCosts` will be a vector of several paths. Maybe use something like `if (all(file.exitst(mif_path_polCosts)))` or `mif_path <- ifelse(file.exitst(mif_path_polCosts), mif_path_polCosts, mif_path)`?

Reply:
@chroetz is right here, I suggest to use the second one. There will always be runs where the policy costs where calculated afterwards and some where the `path_gdx_refpolicycost` run was already finished. Importantly, adding a run without a _adjustedPolicyCosts.mif does then not change the behavior of the rest, which I think is good coding.

Reply:
Thanks for the review! See 9a9f0db.